bucoffea/monojet/definitions.py

```diff
@@ -351,7 +351,7 @@ def setup_candidates(df, cfg):
     taus = taus[ (taus.decaymode) \
                 & (taus.pt > cfg.TAU.CUTS.PT)\
                 & (taus.abseta < cfg.TAU.CUTS.ETA) \
-                & ((taus.iso&2)==2)]
+                & (taus.iso>=2)]
```

Comment:
Is this change necessary? It seems these two statements are equivalent

Reply:
I think you're right, we don't really need the change, I'll undo it